README.md

```diff
@@ -36,6 +36,21 @@ A cleanly architected project should be:
 - _Independent of any external agency_: In fact your business rules simply
   don’t know anything at all about the outside world.
 
+## How to use this library
+
+This project uses [semantic versioning](https://semver.org/) and can be imported as a package in the (engagement-service)[https://github.com/savannahghi/engagement-service] or any other service. 
+
+after importing the package, you can create a new instance from the `usecases`, and use the functionality exposed
+
+## How to deploy changes for this projects
+
+This is a library that uses semantic versioning hence every change made to the library, after approval and release, it must be tagged, then upgrade the (engagement-service)[https://github.com/savannahghi/engagement-service] with the latest tag version.
+
+tagging 
+```sh
+  git tag -m "v0.0.n" "v0.0.n"
+  git push --tags
+ ```
```

Comment:
For now this README will pass. But it needs an MR of it's own. Focus on the moving. Then come back to this.